zebra/interface.c

```diff
@@ -1109,6 +1116,8 @@ void if_down(struct interface *ifp)
 	} else if (IS_ZEBRA_IF_MACVLAN(ifp))
 		zebra_vxlan_macvlan_down(ifp);
 
+	if (zif->es_info.es)
+		zebra_evpn_es_if_oper_state_change(zif, false /*up*/);
```

Comment:
I have given the function arg name as comment - 
void zebra_evpn_es_if_oper_state_change(struct zebra_if *zif, bool up)

In this call up=false; with C++ this would have been a lot easier.